lib/Event/EventOptionsTrait.php

```diff
@@ -19,14 +21,14 @@
     /**
      * This array is used as key value storage for the options.
      *
-     * @var array
+     * @var OptionsResolver
```

Comment:
I've removed the default value, but I am not sure if it is a good idea to change the name of the variable and method, since it feels strange in some places. E.g. if you call `getResolver` just to get the options in a subscriber, and the fact that there is also already a `getOption` method, which should not be renamed. Having two different method names here for more or less the same thing (only the count differs) feels quite strange...